### DISCUSSION TITLE
Added fpm to winget repository

### DISCUSSION THREAD

**Discussion by KiruyaMomochi** [fortran-lang/fpm#744] [Show and tell]
I have added `fpm` to Windows Package Manager repository. 
https://github.com/microsoft/winget-pkgs/pull/78468

For **Windows 10+** users, you can now try to install fpm by

```powershell
winget install -e fpm
```

**Comment by awvwgk** [comment]
Thanks. I'll give it a try next time I'm on my Windows machine. We should also include this in the fpm docs.

**Reply by awvwgk** [reply]
@KiruyaMomochi I made the wiki world-editable. If you don't mind, it would be very helpful to add instructions for updating the winget distribution channel to the wiki at https://github.com/fortran-lang/fpm/wiki/Release-Process.

**Reply by KiruyaMomochi** [reply]
I have added instructions into https://github.com/fortran-lang/fpm/wiki/Release-Process#winget-package.

**Comment by arjenmarkus** [comment]
I was unaware of the winget command, but tried "winget show fpm": two
packages show up, one of them having the ID FortranLang.fpm. Nice :).

Regards,

Arjen

Op ma 5 sep. 2022 om 17:49 schreef Sebastian Ehlert <
***@***.***>:
